src/Documentation/glossary.js

```diff
@@ -42,8 +42,15 @@ export default {
       name: 'Output',
       match: ['output', 'outputs'],
       desc:
-        'A file or a directory that is under DVC control. See `dvc add`,' +
-        ' `dvc run`, `dvc import`, `dvc import-url` commands.'
+        'A file or a directory that is under DVC control. See `dvc add`, ' +
+        '`dvc run`, `dvc import`, `dvc import-url` commands.'
+    },
+    {
+      name: 'External Dependency',
+      match: ['external dependency', 'external dependencies'],
+      desc:
+        'A DVC-file dependency with origin in an external source. See ' +
```

Comment:
it worth specifying some list of external storages, just to give more sense of what is happening here

Reply:
Agree. Added in 634ed43